.travis.yml

```diff
@@ -1,15 +1,22 @@
 language: python
+
 python:
-    - "3.6"
+  - "3.6"
+
 services:
-    - mongodb
+  - mongodb
+
 before_script:
-    - mongo mydb_test --eval 'db.createUser({user: "creep", pwd: "@radiohead.ed", roles: ["readWrite"]});'
-    - cp config.py.example config.py
-    - echo "DB_URI=\"\"" >> config.py
-    - echo "DB_USER=\"creep\"" >> config.py
-    - echo "DB_PASS=\"@radiohead.ed\"" >> config.py
+  - "mongo lcs-db --eval 'db.createUser({user: \"creep\", pwd: \"@radiohead.ed\", roles: [\"readWrite\"]});'"
+  - cp config.py.example config.py
+  - echo "DB_URI=\"127.0.0.1:27017\"" >> config.py
+  - echo "DB_USER=\"creep\"" >> config.py
```

Comment:
i would maybe move the username stuff to a constants file so we can easily change test user

Reply:
Is config.py that sort of file? This bit generates a dummy config so that we don't leak the real one and can test against a db.

Reply:
right but i was saying even the user can be in some constant file (this is up to you)

Reply:
Like a test.config.py? I'll see how it looks.